python/tests/test_bunsen_stu3.py

```diff
@@ -330,15 +365,3 @@ def test_isa_custom(spark_session, bundles):
 
   assert get_current_valuesets(spark_session) == blood_pressure
   assert results.count() == 14
-
-def test_valueset_from_bundle(spark_session):
-  bundles = load_from_directory(spark_session, 'tests/resources/bundles/json', 1)
-
-  vs = extract_entry(spark_session, bundles, 'ValueSet')
-
-  value_sets = create_value_sets(spark_session) \
-    .with_value_sets(vs)
-
-  assert value_sets.get_values("http://hl7.org/fhir/ValueSet/example-extensional", "20150622") \
-           .count() == 4
```

Comment:
Was this test deleted for a reason, or was this a merge error?

Reply:
Good catch. That's definitely should not have been removed. Restored, https://github.com/cerner/bunsen/pull/29/files#diff-6af27a18039c37d475e74904160164f5L342